third_party/abseil-cpp/absl/debugging/symbolize.cc

```diff
@@ -28,7 +28,9 @@
 #define ABSL_INTERNAL_HAVE_SYMBOLIZE_WASM
 #endif
 
-#if defined(ABSL_INTERNAL_HAVE_ELF_SYMBOLIZE)
```

Comment:
Why not unset this define instead?

Reply:
It'll still fall through to other conditionals, like the windows conditional which we don't want